osu.Game/Overlays/Profile/Sections/DrawableUserInfoItem.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using System.Collections.Generic;
+
+namespace osu.Game.Overlays.Profile.Sections
+{
+    public class DrawableUserInfoItem : OsuHoverContainer
```

Comment:
why isn't this called DrawableProfileRow or something? `UserInfoItem` tells me nothing about nothing name i could nothing.